app/routes/_gcn.docs.contributing.project/route.mdx

```diff
@@ -0,0 +1,204 @@
+---
+handle:
+  breadcrumb: Project Board
+---
+
+import { Table } from '@trussworks/react-uswds'
+
+import issueScreenshot from './issue.png'
+import openTicketScreenshot from './openTicket.png'
+import projectBoardScreenshot from './projectBoard.png'
+import ticketMetadata from './ticketMetadata.png'
+
+# GitHub Project Board
+
+This is the [GCN Project Board.](https://github.com/orgs/nasa-gcn/projects/6)
```

Comment:
Would you please turn this into a substantive paragraph? The simplest way that I can see to do that is to remove the `## Philosophy` heading so that the contents of that section becomes the contents of this top-level section, like this:

```md
# GitHub Project Board

We use GitHub Projects to organize our development process.
See our [project board](https://github.com/orgs/nasa-gcn/projects/6) for active issues.
We chose GitHub Projects because...
```